client/mysqlbinlog.cc

```diff
@@ -3143,6 +3133,11 @@ int main(int argc, char** argv)
     start_position= BIN_LOG_HEADER_SIZE;
   }
 
+  if (stop_datetime != MY_TIME_T_MAX &&
+      stop_datetime > last_processed_datetime)
+      warning("Did not reach stop datetime '%s' "
```

Comment:
The `warning` is two spaces too far in. Also the entire string will now fit on one line within `80` chars, now that we have decreased the original indent :)